packages/runtime/container-runtime/src/pendingStateManager.ts

```diff
@@ -44,6 +45,11 @@ interface IPendingFlush {
 
 type IPendingState = IPendingMessage | IPendingFlushMode | IPendingFlush;
 
+interface IPendingSnapshot {
```

Comment:
"snapshot" word has very specific meaning in fluid, maybe change it to something else?
IPendingAction ?